app/presenters/gobierto_data/dataset_presenter.rb

```diff
@@ -0,0 +1,76 @@
+module GobiertoData
+  class DatasetPresenter
+    include ActionView::Helpers::UrlHelper
+
+    attr_reader :site
+
+    def initialize(site)
+      @site = site
+    end
+
+    def build_catalog
+      catalog = {
+        identifier_uri: url_helpers.gobierto_data_root_url(host: site.domain),
+        title:          "dcat catalog for #{site}",
+        description:    "this is catalog published by #{@catalog_identifier_uri} which contains datasets",
+        issued:         site.created_at,
+        modified:       GobiertoData::Dataset.where(site_id: site.id).maximum(:created_at) || site.created_at,
+        languages:      site["configuration_data"]["available_locales"],
+        homepage:       url_helpers.gobierto_data_root_url(host: site.domain),
+        license_url:    "https://opendatacommons.org/licenses/odbl/",
+        datasets:       build_datasets_for_catalog
+      }
+    end
+
+    private
+
+    def build_datasets_for_catalog
+      datasets = []
+      Dataset.where(site_id: site.id).visibles.each do |dataset|
+        datasets <<  build_dataset_for_catalog(dataset)
+      end
+      datasets
+    end
+
+    def build_dataset_for_catalog(dataset)
+      {
+        url: url_helpers.gobierto_data_datasets_url(host: site.domain, id: dataset.slug),
+        title: dataset.name,
+        description: description_custom_field_record(dataset),
+        keywords: [],
+        issued: dataset.created_at,
+        modified: dataset.updated_at,
+        languages: [site_locale],
+        license_url: "https://opendatacommons.org/licenses/odbl/",
```

Comment:
It should read the license custom field of the dataset, right?

Reply:
Yes